packages/sales-api-service/src/schema/contact.schema.js

```diff
@@ -44,7 +45,7 @@ const commonContactSchema = {
   postcode: Joi.when(Joi.ref(DATASOURCE_REF), {
     is: Joi.string().valid(...POCL_TRANSACTION_SOURCES),
     then: Joi.alternatives().conditional('country', {
-      is: Joi.string().valid('GB', 'United Kingdom'),
+      is: Joi.string().valid(...UK_COUNTRIES),
```

Comment:
I'm slightly nervous of the inclusion of 'United Kingdom', 'England', 'Wales', 'Scotland' and 'Northern Ireland', as these aren't ISO countries or our handful of exceptions (GB-ENG, etc). So are we passing these now - because I'm not sure we should?

Reply:
This schema is used for validating pretty much all records that pass through GAFL. I was more worried about leaving them out and causing some unforseen issues and took the lead from the fact that 'United Kingdom' was included, but I guess all that would happen is that their postcode wouldn't be validated, so I'm happy to remove them